projects/documentation/content/guides/adding-component.md

```diff
@@ -48,69 +48,107 @@ must structure our CSS very differently.
 
 ```javascript
-module.exports = {
-    spectrum: 'button',
-    components: [
+// @ts-check
+import {
+    builder,
+    converterFor,
+} from '../../../tasks/process-spectrum-utils.js';
+
+const converter = converterFor('spectrum-Button');
+
+/**
+ * @type { import('../../../tasks/spectrum-css-converter').SpectrumCSSConverter }
+ */
+const config = {
+    conversions: [
         {
-            name: 'button',
-            host: {
-                selector: '.spectrum-Button',
-                shadowSelector: '#button',
-            },
-            focus: '#button',
-            attributes: [
-                {
-                    type: 'boolean',
-                    selector: '.spectrum-Button--quiet',
-                },
-                {
-                    type: 'boolean',
-                    selector: ':disabled',
-                },
+            inPackage: '@spectrum-css/button',
+            outPackage: 'button',
+            fileName: 'button',
+            excludeByComponents: [builder.element('a')],
+            components: [
+                converter.classToHost(),
+                converter.classToAttribute('spectrum-Button--quiet'),
+                converter.classToAttribute('spectrum-Button--emphasized'),
+                converter.classToAttribute('is-disabled', 'disabled'),
+                converter.classToAttribute('is-selected', 'selected'),
+                converter.classToAttribute('is-focused', 'focused'),
+                converter.pseudoToAttribute('disabled', 'disabled'),
+                converter.pseudoToAttribute('active', 'active'),
+                ...converter.enumerateAttributes(
+                    [
+                        ['spectrum-Button--sizeS', 's'],
+                        ['spectrum-Button--sizeM', 'm'],
+                        ['spectrum-Button--sizeL', 'l'],
+                        ['spectrum-Button--sizeXL', 'xl'],
+                    ],
+                    'size'
+                ),
+                ...converter.enumerateAttributes(
+                    [
+                        ['spectrum-Button--accent'],
+                        ['spectrum-Button--primary'],
+                        ['spectrum-Button--secondary'],
+                        ['spectrum-Button--negative'],
+                    ],
+                    'variant'
+                ),
+                ...converter.enumerateAttributes(
+                    [['spectrum-Button--fill'], ['spectrum-Button--outline']],
+                    'treatment'
+                ),
+                ...converter.enumerateAttributes(
+                    [
+                        ['spectrum-Button--staticWhite', 'white'],
+                        ['spectrum-Button--staticBlack', 'black'],
+                    ],
+                    'static'
+                ),
+                converter.classToId('spectrum-Button-label'),
+                converter.classToSlotted('spectrum-Icon', 'icon'),
                 {
```

Comment:
Again, some context to what exactly this selector ends up looking like in the css file would be helpful since this is documentation!

Reply:
Disregard. I think you cover this in the config documentation!